custom/motioncontrollers/MotionController.java

```diff
@@ -256,27 +261,57 @@ public void enable() {
 		}
 		catch (IllegalStateException e) {} // Do not die if the timer is already running
 	}
-
+	
 	/**
 	 * Bypasses the motion controller.
 	 * In some cases, this will still scale by
 	 * a feed forward term of the motion controller.
 	 */
 	public void disable() {
+		if (!isAlive()) {
+			return;
+		}
 		enable = false;
 		task.cancel();
 		timer.purge();
 		task = new MotionControllerTask();
 		setpoint = sensor.pidGet();
 	}
-
+	
 	/**
 	 * Is motion control enabled?
 	 */
 	public boolean isEnabled() {
-		return enable;
+		return enable && isAlive();
 	}
-
+	
+	/**
+	 * Disables the motor controller then kills it.
+	 * Enabling and disabling will no longer be allowed.
+	 * Once killed, a controller must
+	 * be resuscitated for it to work again.
+	 * 
+	 * @see resuscitate()
+	 */
+	public void kill() {
+		disable();
+		alive = false;
+	}
+	
+	/**
+	 * Resuscitates a controller, allowing disable() and enable() to be called again.
+	 */
+	public void resuscitate() {
```

Comment:
perhaps have a single `setOverride` method? or an `enableOverride` and `disableOverride`?